omaha/hammer.bat

```diff
@@ -46,19 +46,19 @@ goto set_env_variables
 :: Change these variables to match the local build environment.
 
 :: Directory where the Go programming language toolchain is installed.
-set GOROOT=C:\go
+set GOROOT=c:\My\Projects\CTheSigns\OmagaTools\go\
```

Comment:
Is this file only for local usage?

Reply:
@KBurov Yes, local usage file :) Reversed